tests/AutoReview/ProjectCodeTest.php

```diff
@@ -716,6 +716,59 @@ public function testAllTestsForShortOpenTagAreHandled(): void
         self::assertSame($testFilesWithShortOpenTag, $phpunitFiles);
     }
 
+    /**
+     * @dataProvider provideTestClassCases
+     *
+     * @param class-string $className
+     */
+    public function testNoDuplicatedMethods(string $className): void
```

Comment:
Other test methods are named with full, grammatically correct sentences. Maybe something like this?


```suggestion
    public function testThatTestMethodsAreNotDuplicated(string $className): void
```